macros/schema_tests/equality.sql

```diff
@@ -13,7 +13,7 @@
 {% set schema = model.schema %}
 {% set model_a_name = model.name %}
 
-{% set dest_columns = adapter.get_columns_in_table(schema, model_a_name) %}
+{% set dest_columns = adapter.get_columns_in_relation(this) %}
```

Comment:
I think we're going to want to use `model` here -- `this` will refer to the currently running model, whereas we want the mode specified as an argument to the macro.